{
  "schema_version": "1.4.0",
  "id": "GHSA-2973-q4qx-mjf5",
  "modified": "2025-04-10T15:31:37Z",
  "published": "2024-09-18T09:30:36Z",
  "aliases": [
    "CVE-2024-46753"
  ],
  "details": "In the Linux kernel, the following vulnerability has been resolved:\n\nbtrfs: handle errors from btrfs_dec_ref() properly\n\nIn walk_up_proc() we BUG_ON(ret) from btrfs_dec_ref().  This is\nincorrect, we have proper error handling here, return the error.",
  "severity": [],
  "affected": [],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2024-46753"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/0e4840ae09f375381167000ce47424818fcbcc7c"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/2c4fe45351e544da4b8f10c74b277117a4fa7869"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/5eb178f373b4f16f3b42d55ff88fc94dd95b93b1"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/67e4ca7ddc67ef949326b4dc404a9678bbe67d72"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/9c8237021b53d52357c0de07a768582fafb2791d"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/a7f16a7a709845855cb5a0e080a52bda5873f9de"
    }
  ],
  "database_specific": {
    "cwe_ids": [],
    "severity": null,
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2024-09-18T08:15:04Z"
  }
}